File: programmers/js_src/12932.js
// 자연수 뒤집어 배열로 만들기

function solution(n) {
  var answer = [];
  var strNum = String(n);
  
  // for (let i of strNum) {
  //     answer.unshift(+i);
  // }
  
  strNum.split('').reverse().forEach(str => answer.push(+str));
  
  return answer;
}

File: programmers/py_src/12932.py
# 자연수 뒤집어 배열로 만들기

def solution(n):
    answer = []
    strNum = str(n)
    
    for i in strNum:
        answer.append(int(i))
        
    answer.reverse()
    
    return answer
    
# 다른 풀이
# def solution(n):
#     return list(map(int, reversed(str(n))))